lib/console_io.rb

```diff
@@ -0,0 +1,38 @@
+class ConsoleIO
```

Comment:
I'm not sure how valuable this would be, but I think it would be cool here to inherit from some base `IO` class or something that acts as an interface. That way if you pass something into the `App` class, and it doesn't confine to this interface you've built, the base class would raise a "Method Not Implemented" error.